res/values-fr/strings.xml

```diff
@@ -58,13 +58,13 @@
     <string name="SERVER">"SERVEUR"</string>
     <string name="Joining_game_in_">"Rejoindre la partie "</string>
     <string name="Game_Full">"Partie complète"</string>
-    <string name="US_East">"Amérique Centrale"</string>
-    <string name="US_West">"Amérique du Nord"</string>
+    <string name="US_East">"Amérique de l'ouest"</string>
+    <string name="US_West">"Amérique de l'est"</string>
```

Comment:
The server names at lines 61 and 62 must not be redone back to previous names, as they were also changed in English. Proof is at English file `res/values` and in-game English. Please return the original names here

Reply:
What do i have to do now sir ?

Reply:
You can edit the file through this same pull request (consequently submitting another commit here) and return the names as they were

Reply:
Oh okay i can also add some errors or i need to make a new request ? And maybe find others dots, because i didn’t checked everything tbh. And i want the translator tag but i don’t know how many words i need to translate right.

Reply:
You just don't need to fix every line, but those you think that are really in need. You don't need to submit another PR (Pull Request), as I already said, you can just access this PR, then submit another commit here with the fixes

Reply:
Okay well thank you i’ll do it as soon as i can.